Misc/NEWS.d/next/Core and Builtins/2020-05-27-16-08-16.bpo-38605.rcs2uK.rst

```diff
@@ -0,0 +1 @@
+Enable postponed evaluation of annotations (:pep:`563`) by default.
```

Comment:
Given how monumental of an effort this PR is proving to be I think we should elaborate a bit here. Maybe just
```suggestion
Enable ``from __future__ import annotations`` (:pep:`563`) by default.
The values found in `__annotations__` dicts are now strings, e.g.
``{"x": "int"}`` instead of ``{"x": int}``.
```